hw/ip/rv_core_ibex/data/rv_core_ibex.hjson

```diff
@@ -399,6 +399,22 @@
       expose:  "true"
     },
 
+    { name:    "DmBaseAddr"
+      type:    "int unsigned"
+      default: "437321728" //"32'h1A110000"
+      desc:    "Base address of Debug Module"
+      local:   "false"
```

Comment:
Maybe just make a local parameter? I don't think that should be overwritten, and if, it might cause some security issue?

Reply:
As the related parameters `DmHaltAddr` and `DmExceptionAddr` are also non-local and can thus be overwritten, I think the DM base address and the DM address mask also need to be non-local. Otherwise a top could overwrite the halt address, for example, to fall outside of the DM address range, which couldn't be changed in the top config. Does that make sense?

What security issue did you have in mind? These are elaboration-time constants, so protecting their integrity is part of protecting the integrity of the code base. The constants are not confidential.